README.md

```diff
@@ -16,10 +16,17 @@ $ sudo make install
 ## Configuration
 
 ```
-auth sufficient pam_oauth2.so <tokeninfo url> <login field> key1=value2 key2=value2
+auth sufficient pam_oauth2.so <tokeninfo url> <login field> [POST] [:<Authz payload>] key1=value2 key2=value2
 account sufficient pam_oauth2.so
 ```
 
+Optional parameter `POST` indicates to use POST method when sending request to the `<tokeninfo url>`. This also adds 
+`Content-Type: application/x-www-form-urlencoded` header and moves CGI parameters from `<tokeninfo url>` into request body.
+If this parameter is omitted, the GET method is used.
+
+Optional parameter `:<Authz payload>` specify a payload for `Authorization` HTTP header, e.g. `:dXNlcjpwYXNzd29yZA==` will result 
```

Comment:
Hmm. Should it maybe be more explicit?
Something like:
```
[Authorization: Basic dXNlcjpwYXNzd29yZA=]
```
So that it will be possible to also use:
```
[Authorization: Bearer xyzabcd]
```
And check for `Authorization:` prefix instead of just `':'`.

According to the pam docs square brackets are needed if argument include space.

Reply:
Sorry, I forgot about other authorization schemes that can be used :)
Perhaps if we go with the following syntax:
```
:<authorisation scheme>:<authorisation payload>
```
e.g.
```
:Bearer:xyzabcd
```
it would be more compact and nice looking. The square brackets would be needed only if the authorisation payload includes spaces.

Reply:
IMO, it is better to be explicit.
Someone who will have to maintain the configuration would easier understand what is happening just from the first sight if the argument looks exactly as in HTTP header.

Reply:
The person who have to maintain the configuration will rather be following a defined operational procedure, anyone else will just read the docs. Making it looks like a HTTP header may give a false impression that any other HTTP header can be inserted which is not true.
I have added support for authorisation scheme. Feel free to change it any way you like.